src/components/NavigationAccount.vue

```diff
@@ -129,16 +129,32 @@ export default {
 		},
 		deleteAccount() {
 			const id = this.account.id
+			OC.dialogs.confirmDestructive(
+				t('mail', 'Fully delete {accountId}\'s account', { accountId: this.account }),
```

Comment:
This should have some more info, like:
> The account and cached email data will be removed from Nextcloud, but not from your email provider.
```suggestion
				t('mail', 'The account and cached email data will be removed from Nextcloud, but not from your email provider.'),
```

Reply:
No need to include the accountId in the body text here, as it’s on the button. :)